src/App.js

```diff
@@ -1,9 +1,28 @@
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Home from "./components/home";
+import Navbar from "./components/navbar";
+import People from "./components/people";
+import Movies from "./components/movies";
+import Locations from "./components/locations";
+
+
 function App() {
+
   return (
     <div className="app">
-      <h1>Welcome to GhibliApp</h1>
+      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css"></link>
+      <BrowserRouter>
+        <Navbar/>
+        <Routes>
+          <Route style={{color:"red"}} path="/homes" element={<Home/>} />
```

Comment:
For the Home component to be set up properly, gotta have the path point to `/`, so that when you land on the root path, you see the home component